src/widgets/vitals/vitals-biometrics-form/vitals-biometrics-input.component.tsx

```diff
@@ -0,0 +1,84 @@
+import { NumberInput, TextArea, TextInput } from "carbon-components-react";
+import { debounce } from "lodash-es";
+import React, { Fragment } from "react";
+import styles from "./vitals-biometrics-input.component.scss";
+
+interface VitalsBiometricInputProps {
+  title: string;
+  onInputChange(evnt): void;
+  textFields: Array<{
+    name: string;
+    separator?: string;
+    type?: string | "text";
+    value: number | string;
+    className?: string;
+  }>;
+  unitSymbol?: string;
+  textFieldWidth?: string;
+  textFieldStyles?: React.CSSProperties;
+  placeholder?: string;
+  disabled?: boolean;
+  isValidRange?: boolean;
+}
+
+const VitalsBiometricInput: React.FC<VitalsBiometricInputProps> = ({
+  title,
+  onInputChange,
+  textFields,
+  unitSymbol,
+  textFieldStyles,
+  textFieldWidth,
+  placeholder,
+  disabled,
+  isValidRange: isValid
+}) => {
+  return (
+    <div className={styles.inputContainer} style={{ width: textFieldWidth }}>
+      <p className={styles.vitalsBiometricInputLabel01}>{title}</p>
+      <div
+        className={`${styles.textInputContainer} ${disabled &&
+          styles.disableInput} ${isValid && styles.danger}`}
```

Comment:
```suggestion
          styles.disableInput} ${!inputIsValid && styles.danger}`}
```

Reply:
This is the change corresponding to the backwards function implementation. Right now the logic says "if the input is valid, mark it with danger." What we want is "if the input is not valid, mark it with danger."